cypress-io/cypress
# fix: Remove erroneous <br> tags from error messages

- Closes https://github.com/cypress-io/cypress/issues/28452

### Additional details

There was a regression introduced in 12.4.0 with [this PR](https://github.com/cypress-io/cypress/pull/25467), where all of our error messages were formatting to include `<br>` tags when there was a new line in the error message.

The original PR did not run Percy checks since it was an outside contributor, so this visual change went unnoticed.

This adds an exception to the formatting logic when the message is of type: error in order to maintain the functionality of the originally intended PR to have line breaks in Cypress.log messages.

### Steps to test

A test was added to cover this in the test_errors.cy.ts, we should also see Percy snapshots update.

### How has the user experience changed?

#### Before

![image (4)](https://github.com/cypress-io/cypress/assets/1271364/ab2efc54-0274-4781-abdf-339fde3a9641)
![image (5)](https://github.com/cypress-io/cypress/assets/1271364/f1000b28-b248-4953-bfc3-4343810d8c4c)

#### After

![Screen Shot 2023-12-11 at 11 22 18 AM](https://github.com/cypress-io/cypress/assets/1271364/f99fef0a-8e42-4748-b649-1d1a69dbf8a6)

![Screen Shot 2023-12-11 at 12 12 39 PM](https://github.com/cypress-io/cypress/assets/1271364/5b63f8c7-19a9-4725-9503-44f4c7905573)

### PR Tasks


- [NA] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [NA] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
